src/main/java/com/google/sps/servlets/UserActionServlet.java

```diff
@@ -14,35 +14,37 @@
 
 package com.google.sps.servlets;
 
-import com.google.appengine.api.users.UserService;
-import com.google.appengine.api.users.UserServiceFactory;
 import com.google.gson.Gson;
-import com.google.gson.JsonObject;
-import com.google.sps.util.UserHelper;
+import com.google.sps.data.User;
 import java.io.IOException;
-import java.io.PrintWriter;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+/**
+ * provides get method for adding a recipe ID to one of the three User recipe lists.
+ * request parameters: "destination", "inputID"
+ * "destination" may be one of three user recipe list: "cookbook", "userRecipes", "planner".
+ * "inputID" must be the recipe ID to be added.
+ */
+@WebServlet("/user-action")
+public class UserActionServlet extends HttpServlet {
 
-@WebServlet("/user-status")
-public class UserStatusServlet extends HttpServlet {
-
-  // Response: returns some user information JSON from the current user's sign-in API profile.
-  // Data: "isUserLoggedIn" : boolean
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("application/json");
+    String destination = (String) request.getParameter("destination");
+    long ID = Long.parse((String) request.getParameter("inputID"));
+    
+    // Must be changed to implement distinct users with sign-in API.
+    User user = User.getTestUser();
 
-    JsonObject userInfo = new JsonObject();
-    userInfo.addProperty("isUserLoggedIn", UserHelper.isUserLoggedIn());
-    if (UserHelper.isUserLoggedIn()) {
-        System.out.println("User is logged in");
-        // other properties may be added in the future.
+    if (destination == "cookbook") {
```

Comment:
Use .equals() to compare strings. Here's a [stackoverflow](https://stackoverflow.com/questions/7520432/what-is-the-difference-between-and-equals-in-java#:~:text=The%20equals()%20method%20compares,same%20String%20instance%20or%20not.&text=On%20the%20other%20hand%2C%20the,to%20the%20same%20String%20instance), the tl;dr is that .equals() compares the values, while == compares the object references